security/src/main/java/io/micronaut/security/x509/X509Authentication.java

```diff
@@ -27,16 +28,18 @@
  */
 public class X509Authentication extends ServerAuthentication {
 
+    @NonNull
```

Comment:
is this needed (and checked)? I would expect the annotation on the constructor arg and getter to be sufficient

Reply:
I typically annotate the field and the generate constructor and getter.